bot/cogs/modlog.py

```diff
@@ -670,7 +679,12 @@ def ignore(self, event: Event, *items: int):
         except NotFound:  # Was deleted before we got the event
             return
 
-        if message.guild.id != GuildConstant.id or message.channel.id in GuildConstant.ignored or message.author.bot:
+        if (
+                not message.guild
+                or message.guild.id != GuildConstant.id
+                or message.channel.id in GuildConstant.ignored
+                or message.author.bot
```

Comment:
These are indented too far.

That being said, this isn't exactly pretty either. Given this exact check happens twice, maybe it could be turned into a function and in there we may be able to make it nicer?

Reply:
That's how PyCharm auto-indented them...